app/src/main/res/layout/drag_drop_test_activity.xml

```diff
@@ -1,7 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.recyclerview.widget.RecyclerView xmlns:android="http://schemas.android.com/apk/res/android"
-  xmlns:app="http://schemas.android.com/apk/res-auto"
-  android:id="@+id/drag_drop_recycler_view"
-  android:layout_width="match_parent"
-  android:layout_height="wrap_content"
-  app:layoutManager="androidx.recyclerview.widget.LinearLayoutManager" />
+<layout xmlns:android="http://schemas.android.com/apk/res/android">
+
+  <fragment
```

Comment:
This should be a FrameLayout since we're defining the fragment in code, not in XML.